Simplified/NYPLAnnotations.swift

```diff
@@ -12,177 +12,158 @@ final class NYPLAnnotations: NSObject {
     _ error: NSError?) -> ()) {
-    
-        //FIXME: this network code needs to be tested before being re-implemented. Alamofire was removed. Remove this comment after turning annotations/sync back on.
-    
-//    if (NYPLAccount.shared().hasBarcodeAndPIN())
-//    {
-//      if book.annotationsURL != nil {
-//        
-//        var request = URLRequest(url: book.annotationsURL)
-//        request.httpMethod = "GET"
-//        
-//        for (headerKey, headerValue) in NYPLAnnotations.headers {
-//          request.setValue(headerValue, forHTTPHeaderField: headerKey)
-//        }
-//        
-//        let dataTask = URLSession.shared.dataTask(with: request) { (data, response, error) in
-//          
-//          if error != nil {
-//            completionHandler(nil, error as? NSError)
-//          } else {
-//            
-//            let jsonData: [String:Any]?
-//            do {
-//              jsonData = try JSONSerialization.jsonObject(with: data!, options: []) as? [String:Any]
-//            } catch {
-//              print("JSON could not be created from data.")
-//              completionHandler(nil, nil)
-//              return
-//            }
-//            
-//            if let json = jsonData {
-//              let total:Int = json["total"] as! Int
-//              if total > 0
-//              {
-//                let first = json["first"] as! [String:AnyObject]
-//                let items = first["items"] as! [AnyObject]
-//                for item in items
-//                {
-//                  let target = item["target"] as! [String:AnyObject]
-//                  let source = target["source"] as! String
-//                  if source == book.identifier
-//                  {
-//                    let selector = target["selector"] as! [String:AnyObject]
-//                    let value = selector["value"] as! String
-//                    
-//                    completionHandler(value as String!, error as? NSError)
-//                    print(value)
-//                  }
-//                }
-//              }
-//            } else {
-//              print("Unexpected data returned as JSON")
-//              completionHandler(nil, nil)
-//            }
-//          }
-//        }
-//        dataTask.resume()
-//      }
-//    }
+       
+    if (NYPLAccount.shared().hasBarcodeAndPIN())
+    {
+      if book.annotationsURL != nil {
+        
+        var request = URLRequest(url: book.annotationsURL)
+        request.httpMethod = "GET"
+        
+        for (headerKey, headerValue) in NYPLAnnotations.headers {
+          request.setValue(headerValue, forHTTPHeaderField: headerKey)
+        }
+        
+        let dataTask = URLSession.shared.dataTask(with: request) { (data, response, error) in
+          
+          if error != nil {
+            completionHandler(nil, error as? NSError)
+          } else {
```

Comment:
@winniequinn yes a return should be called actually after any completion handler. looking at the code i am also removing the error, as it is not really needed so the completion handler will just have the result or nil